src/CSV.jl

```diff
@@ -425,6 +428,31 @@ function file(source,
     else
         typecodes = TypeCode[T for _ = 1:ncols]
     end
+    # set any unselected columns to typecode USER | MISSING
+    todrop = Int[]
+    if select !== nothing
+        if select isa AbstractVector{Int}
+            foreach(1:ncols) do i
```

Comment:
Why not just:
```suggestion
            for i in 1:ncols
```
Same below.